pkg/operator/ceph/nfs/nfs.go

```diff
@@ -276,6 +298,14 @@ func validateGanesha(context *clusterd.Context, clusterInfo *cephclient.ClusterI
 	// The existence of the pool provided in n.Spec.RADOS.Pool is necessary otherwise addRADOSConfigFile() will fail
 	_, err := cephclient.GetPoolDetails(context, clusterInfo, n.Spec.RADOS.Pool)
 	if err != nil {
+		if strings.Contains(err.Error(), "unrecognized pool") {
+			if clusterInfo.CephVersion.IsAtLeastPacific() {
```

Comment:
Should this be comparing against v16.2.6 instead of just v16?

Reply:
If the version is above pacific we expect a default pool name so I should create the corresponding pool if it doesn't exist, `n.Spec.RADOS.Pool` contains the name of the default pool from when I set it earlier in the code,